server/index.js

```diff
@@ -61,12 +61,13 @@ export default class Server {
       parsedUrl.query = parseQs(parsedUrl.query)
     }
 
+    res.statusCode = 200
```

Comment:
Ah nvm, this is when handling the route before we pass it off to the page component. 👍